spec/formatting.md

```diff
@@ -191,6 +191,48 @@ and different implementations MAY choose to perform different levels of resoluti
 
 _Expressions_ are used in _declarations_ and _patterns_.
 _Markup_ is only used in _patterns_.
+_Options_ are used in _expressions_ and _markup_.
+
+#### Option Resolution
```

Comment:
The section should probably go either immediately before or after Markup Resolution, rather than being the first one. These sections have a top-down-ish order, and we start from "Expression and Markup Resolution".

Reply:
Your current proposal has the sections ordered as:
1. Option Resolution
2. Expression Resolution
3. Markup Resolution
4. Fallback Resolution

I think the order should be instead:
1. Expression Resolution
2. Markup Resolution
3. Option Resolution
4. Fallback Resolution